inference/huggingface/text-generation/README.md

```diff
@@ -91,8 +91,9 @@ The DSPipeline class helps to load the model and run inference on it, given thes
 # DeepSpeed HuggingFace Compare
 
 The ds-hf-compare script can be used to compare the text generated outputs of DeepSpeed with kernel injection and HuggingFace inference of a model with the same parameters on a single GPU.
+(p.s. Kernel injection is not supported on CPUs, so you need to disable kernel injection by setting "--use_kernel" as "False" when running on CPUs.)
 
 ## Usage
 Examples can be run as follows:
-<pre>deepspeed --num_gpus 1 ds-hf-compare.py --model [model name/path] --dtype [data type] --num_inputs [number of test inputs] --print_outputs
+<pre>deepspeed --num_gpus 1 ds-hf-compare.py --model [model name/path] --dtype [data type] --num_inputs [number of test inputs] --print_outputs --use_kernel[enable kernel injection or not True/False]
```

Comment:
Nit: I think we can remove the ` or not True/False` part here, since `use_kernel` will act as an enabling argument if provided.

Reply:
Solved